main/src/org/destinationsol/game/planet/PlanetTiles.java

```diff
@@ -34,7 +34,7 @@
     private final Map<SurfaceDirection, Map<SurfaceDirection, List<Tile>>> myGroundTiles;
 
     public PlanetTiles(TextureManager textureManager, String groundFolder, FileHandle configFile) {
-        myGroundTiles = new HashMap<SurfaceDirection, Map<SurfaceDirection, List<Tile>>>();
+        myGroundTiles = new HashMap<>();
```

Comment:
Same reason as above, exact same functionality with cleaner code.
Aaaaand I hate the checkstyle warnings.